README.md

```diff
@@ -1,25 +1,108 @@
- 6. Create a designeated init method for FISLocation.
- 6. Modify the `searchForLocationName:inLocations:` method to search an `NSArray` of `FISLocation` objects.
- 7. Create a new class called `FISTrivia`. This class should have the following properties:
+In the last *Location Trivia* lab you dealt with a representation of locations using `NSDictionary`. Now, increase the power of your programming by storing this information in custom classes instead!
+
+*  Fork and clone this lab.
+
+*  Open the `locationTrivia-objects.xcworkspace` file.
+
+### I. Create the Custom Classes
+
+Create the class files for `FISLocation` that inherits from `NSObject`. This class should have four properties:
+
+  * an `NSString` called `name`,
+  * a `CGFloat` called `latitude`,
+  * a `CGFloat` called `longitude`, and
+  * an `NSMutableArray` called `trivia`.
+
+Create the class files for `FISTrivium` that inherits from `NSObject`. This class should have two properties:
+
+  * an `NSString` called `content`, and
+  * an `NSUInteger` called `likes`.
+
+On the `FISAppDelegate` class add a public property:
+
+  * an `NSMutableArray` called `locations`.
+
+### II. Declare Methods and Define Initial Implementations
+
+On the `FISAppDelegate` class, declare three methods:
+
+  * `allLocationNames` that takes no arguments and returns an `NSArray`,
+  * `locationWithName:` that takes one `NSString` argument called `name` and returns a `FISLocation` object, and
+  * `locationsNearLatitude:longitude:margin:` that takes three `CGFloat` arguments called `latitude`, `longitude`, and `margin`, and returns an `NSArray`.
+
+Define these methods to return `nil`.
+
+On the `FISLocation` class, declare five methods:
+
+  * `init` — the default initializer that will be overridden,
+  * `initWithName:latitude:longitude:` — a designated initializer,
+  * `stringByTruncatingNameToLength:` which takes one `NSUInteger` argument called `length` and returns an `NSString`,
+  * `validateLocationData` which takes no arguments and returns a `BOOL`, and
+  * `triviumWithMostLikes` which takes no arguments and returns a `FISTrivium` object.
+
+Define the initializers to assign to `self` a call of `[super init]` and return `self`.
+
+Define the other three methods to return `nil`.
+
+On the `FISTrivium` class, declare two initializers:
+
+  * `init` — the default initializer that will be overridden, and
+  * `initWithContent:likes:` — a designated initializer.
+
+Define the initializers to assign to `self` a call of `[super init]` and return `self`.
+
+Run the tests with `⌘` `U` to see that they fail.
+
+### III. Build the `FISTrivium` Class
+
+1. Define `FISTrivium`'s designated initializer.
+
+2. Override `FISTrivium`'s default initializer to call the designated initializer. Have it pass in an empty string (`@""`) for the `content` argument, and `0` (zero) for the `likes` argument.
```

Comment:
I kind of like the idea of being a little less direct about telling them what to do. Instead describe the goal:
"Override `FISTrivium`'s default initializer to call the designated initializer. Default instances of `FISTrivium` should have an empty string for their `content` and no `likes`."